source/replication/site-replication-overview.rst

```diff
@@ -0,0 +1,532 @@
+.. _minio-site-replication-overview:
+
+=========================
+Site Replication Overview
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Site replication configures multiple independent MinIO deployments as a cluster of replicas called peer sites.
+
+Site replication assumes the use of either the included MinIO identity provider (IDP) *or* an external IDP.
+All configured deployments must use the same IDP.
+Deployments using an external IDP must use the same configuration across sites.
+
+Overview
+--------
+
+.. _minio-site-replication-what-replicates:
+
+What Replicates Across All Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-replicates
+   :end-before: end-mc-admin-replicate-what-replicates
+
+What Does Not Replicate Across Sites
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /includes/common-replication.rst
+   :start-after: start-mc-admin-replicate-what-does-not-replicate
+   :end-before: end-mc-admin-replicate-what-does-not-replicate
+
+
+Initial Site Replication Process
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After enabling site replication, identity and access management (IAM) settings sync in the following order:
+
+.. tab-set::
+
+   .. tab-item:: MinIO IDP
+
+      #. Policies
+      #. User accounts (for local users)
+      #. Groups
+      #. Service accounts
+         
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: OIDC
+
+      #. Policies
+      #. Service accounts
+         
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+   .. tab-item:: LDAP
+
+      #. Policies
+      #. Groups
+      #. Service accounts
+
+         Only service accounts related to user accounts with valid policies for the MinIO deployment.
+         Service accounts for ``root`` do not sync.
+
+      #. Policy mapping for synced user accounts
+      #. Policy mapping for `Security Token Service (STS) users <https://docs.min.io/docs/minio-sts-quickstart-guide.html>`
+
+
```

Comment:
I think we should add something like:

```suggestion
After initial synchronization of data completes, MinIO continually replicates and synchronizes :ref:`replication-eligible changes <>` as they occur.
```

With a label to the section on "What replicates"

Reply:
Slight tweak, but yes.